### ISSUE TITLE
duplicate git repositories

### ISSUE THREAD

**Issue by tdons** [keybase/keybase-issues#3157]
## Problem
git repositories are *not* unique.

## Reproduce
The below code worked on my first try:
```sh
$ { keybase git create test & } ; keybase git create test &
```

## Actual result
```sh
Repo created! You can clone it with:
  git clone keybase://private/tdons/test
Repo created! You can clone it with:
  git clone keybase://private/tdons/test
$ keybase git list | grep test
  test                      keybase://private/tdons/test
  test                      keybase://private/tdons/test
```

## Expected result
```sh
$ keybase git create test
▶ ERROR A repo named "test" already exists.
```

## Additional information

Deleting the duplicate repositories is possible by issuing `keybase git delete` twice:
```sh
$ keybase git delete test
Deletion is permanent. Are you sure you want to delete "test"? (type 'YES' to confirm): yes
Repo deleted!
$ keybase git list | grep test
  test                      keybase://private/tdons/test
$ keybase git delete test
Deletion is permanent. Are you sure you want to delete "test"? (type 'YES' to confirm): yes
▶ WARNING Git repo doesn't exist. Deleting metadata anyway.
Repo deleted!
$ keybase git list | grep test
$
```

**Comment by strib** [comment]
A fix and a test is in review: keybase/kbfs#1446.  Thanks for the report!

**Comment by strib** [comment]
Fixed in master, should be working in the next release.
